modular_zubbers/code/datums/quirks/positive_quirks/venemous_bite/venemous_bite_quirk.dm

```diff
@@ -0,0 +1,100 @@
+/// Format: (reagent typepath -> list(amount to inject per bite, cooldown))
+GLOBAL_LIST_INIT(venomous_bite_choice_specs, list(
```

Comment:
This could just be a `var/static/list` attached to the quirk type

Reply:
IFAIK This doesnt allow it to be externally referenced, which causes issues with both preference and the action itself

Reply:
(Both the pref and the action directly reference this list)

Reply:
> IFAIK This doesnt allow it to be externally referenced, which causes issues with both preference and the action itself

https://www.byond.com/docs/ref/#/operator/::